tests/torch/test_hybrid_converter.py

```diff
@@ -40,6 +40,8 @@ def run_hybrid_llm_test(
     # Multi-parameter strategy is used in order to speed-up the FHE executions
     configuration = Configuration(
         single_precision=False,
+        compress_input_ciphertexts=True,
```

Comment:
these circuits can be fully leveled or not, always good to test with default compilation parameters

Reply:
so why not also enabling key compression ?

Reply:
it is enabled by the QM compilation (the env variable defaults to 1)

Reply:
so is `compress_input_ciphertexts` then right ? imo either we put both here or we revert this small change 🤔